client-app/pages/reset-password/reset-password.vue

```diff
@@ -1,27 +1,30 @@
 <template>
   <TwoColumn class="max-w-screen-xl">
     <template #left>
-      <h1 class="uppercase tracking-wide text-3xl lg:text-4xl font-bold mb-8 lg:mb-9 lg:mt-6">reset your password</h1>
+      <h1
+        class="uppercase tracking-wide text-3xl lg:text-4xl font-bold mb-8 lg:mb-9 lg:mt-6"
+        v-t="'pages.reset-password.header'"
```

Comment:
Why "reset-password" is dashed? All previous similar page keys were renamed to underscore notation.